docs/patch-metadata.md

```diff
@@ -19,6 +19,13 @@ These metadata numbers can be taken from consulting appropriate MOS Notes, such
 * Database 18c Proactive Patch Information (Doc ID 2369376.1)
 * Database 12.2.0.1 Proactive Patch Information (Doc ID 2285557.1)
 
+The md5sum can be determined by listing the file once in a GCS bucket:
+
+```
+$ gsutil ls -L gs://example-buckeet/p32578973_190000_Linux-x86-64.zip | grep md5
```

Comment:
Not that it matters, but a minor typo in `buckeet`

Reply:
Fixed